ddl/ddl.go

```diff
@@ -1329,6 +1329,12 @@ type RecoverSchemaInfo struct {
 // It should be called before any DDL that could break data consistency.
 // This provides a safe window for async commit and 1PC to commit with an old schema.
 func delayForAsyncCommit() {
+	if variable.EnableMDL.Load() {
```

Comment:
Is it possible to switch off MDL during an ongoing DDL and break some constraints?

Reply:
No, we forbid to switch MDL if there is any DDL running